src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,34 +1,82 @@
 package edu.kis.vh.nursery;
 
+/**
+ * DefaultCountingOutRhymer is class, which is base implementation for rhymers.
+ *
+ * It can count in numbers up to pre-defined value, and count them out in stack-similiar way.
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int MAX_NUMBERS_COUNT = 12;
+    private static final int COUNT_OUT_FINISHED = -1;
+    private int[] numbers = new int[MAX_NUMBERS_COUNT];
 
-    public int total = -1;
+    /**
+     * Total amount of numbers amassed in rhymer.
+     */
+    public int total = COUNT_OUT_FINISHED;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)